sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -230,12 +231,12 @@ def _allocate_parameter_mem(self, n_classes, n_features, coef_init=None,
 
         # initialize average parameters
         if self.average > 0:
-            self.standard_coef_ = self.coef_
-            self.standard_intercept_ = self.intercept_
-            self.average_coef_ = np.zeros(self.coef_.shape,
+            self._standard_coef_ = self.coef_
+            self._standard_intercept_ = self.intercept_
+            self._average_coef_ = np.zeros(self.coef_.shape,
```

Comment:
The final underscore is not necessary.

Reply:
This has not been addressed. I mean please rename the new attributes ``_standard_coef``, ``_standard_intercept``, ... It makes the code simpler to read.